R/app.R

```diff
@@ -0,0 +1,289 @@
+library(shiny)
+library(shinydashboard)
+library(DT)
+library(tidyverse)
+library(tidyquant)
+library(quantmod)
+library(crypto2)
+library(plotly)
+library(gridExtra)
+
+
+# ------------------------------------------------------
+# LOAD DATA
+#-------------------------------------------------------
+# Get dataframe with historical crypto data
+top_10_crypto <- crypto_history(limit=10,
+                                start_date='20210601',
+                                finalWait = FALSE)
+
+# Convert time_open to date
+top_10_crypto$time_open<-as.Date(top_10_crypto$time_open)
+
+
+# Extract names of top 10 cryptos
+top_10_crypto_names <- unique(top_10_crypto$name)
+  
+
+
+# ------------------------------------------------------
+# UI
+#-------------------------------------------------------
+ui <- dashboardPage(
```

Comment:
Maybe it would be a good idea to move ui also to a different R file? What do you think?

Reply:
Let's do it in a separate pull request.